homework/index.js

```diff
@@ -30,21 +31,38 @@
   }
 
   function renderRepoDetails(repo, ul) {
-    createAndAppend('li', ul, { text: repo.name });
+    createAndAppend('li', ul, { 
+      text: `
+      <div class="repoInfo">
+      <p class="repoLine"><span class="bold">Repository: </span> <a href=${repo.html_url}>${repo.name}</a></p>
+      <p class="repoLine"><span class="bold">Description: </span>${repo.description}</p>
+      <p class="repoLine"><span class="bold">Forks: </span> ${repo.forks}</p>
+      <p class="repoLine"><span class="bold">Updated: </span> ${repo.updated_at}</p>
+    </div>
+    `});
   }
 
   function main(url) {
     fetchJSON(url, (err, repos) => {
       const root = document.getElementById('root');
       if (err) {
-        createAndAppend('div', root, {
-          text: err.message,
-          class: 'alert-error',
-        });
+        createAndAppend('div', root, { 
+          text: `          
+          <h3 class="heading">HYF Repositories</h3>
+          <p class="error">${err.message}</p>  `,  
+          class: "alert-error"         
+      }
+        );
         return;
       }
+      const h3 = createAndAppend('h3', root, {
+        text: 'HYF Repositories',
+        class: 'heading'
+      });
       const ul = createAndAppend('ul', root);
-      repos.forEach(repo => renderRepoDetails(repo, ul));
+      repos.slice(0, 10).sort((a, b) =>a.name
+      .localeCompare(b.name, 'en', { numeric: true }, { sensitivity: 'base' }))
+      .forEach(repo => renderRepoDetails(repo, ul));
```

Comment:
+1 for chaining methods
But chained methods are easier to read if you indent them

```suggestion
      repos
          .slice(0, 10)
          .sort((a, b) =>a.name.localeCompare(b.name, 'en', { numeric: true }, { sensitivity: 'base' }))
          .forEach(repo => renderRepoDetails(repo, ul));
```